op-guide/migration.md

```diff
@@ -201,6 +198,29 @@ mysql> select * from t2;
 +----+------+
 ```
 
+### 3. Best practice
+In order to migrate data quickly (especially huge amount of data), you can refer to the following recommendations.
+
+- make dump data file as small as possible, it is best not to exceed 64M (parameter -F 64)
+- loader's parameter -t can be evaluated according to the number and load of tikv instances. For example, there are three tikv instances, -t can be set to 3 * (1 ~ n); while load of tikv is too high and the loader's log appears a large number of `backoffer.maxSleep 15000ms is exceeded`, it's better to decrease -t ; otherwise increase it.
+
+### A sample, and related configuration
+
+ - the total amount of dump files are 214G, single table with 8 columns, 2 billion rows
+ - cluster topology
+    - TIKV * 12 [4 machines, 3 tikv instances per macine]
+    - TIDB * 4
+    - PD * 3
+ - machine configuration
+    - CPU: Intel Xeon E5-2670 v3 @ 2.30GHz
+    - 48 vCPU [2 x 12 physical cores]
+    - Memory: 128G
+    - Disk: sda [raid 10, 300G] sdb[raid 5, 2T]
+    - OS: CentOS 7.2
+ - mydumper's -F is set to 16, loader's -t is set to 64
```

Comment:
The `F` parameter of mydumper is set to `16` and the `-t` parameter of loader is set to `64`.